contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -39,7 +39,7 @@ install() {
 	dracut_install @sbindir@/zfs
 	dracut_install @sbindir@/zpool
 	# Include libgcc_s.so.1 to workaround zfsonlinux/zfs#4749
-	if type gcc-config 2>&1 1>/dev/null; then
+	if command -v gcc-config 2>&1 1>/dev/null; then
 		dracut_install "/usr/lib/gcc/$(s=$(gcc-config -c); echo ${s%-*}/${s##*-})/libgcc_s.so.1"
 	else
 		dracut_install /usr/lib/gcc/*/*/libgcc_s.so.1
```

Comment:
@behlendorf Do you mean that libpthread.so will attempt to dlopen() libgcc_so.1 from /lib64/libgcc_s.so.1 rather than `/usr/lib/gcc/*/*/libgcc_s.so.1`? Otherwise, this should be fine as long as `dracut_install` follows the symlink.

As far as I can tell, libgcc_s.so.1 is a hack that moved low level logic into GCC so that glibc would not block development of GCC. `/usr/lib/gcc/*/*/libgcc_s.so.1` is the upstream location for this so that multiple GCC versions can co-exist with one another and libpthread.so is supposed to look there. Redhat appears to have moved libgcc.so.1 into /lib64, but this implies that they will either never update GCC or they intend to update it themselves if they add support for a GCC that needs newer functionality. Presumably, the changes to libgcc_s.so.1 are backward compatible and Redhat have symlinks in place at the location upstream designated because they expect software to expect to find it there.